api/handlers_services.go

```diff
@@ -77,16 +77,6 @@ func (s *server) ServiceDeleteHandler(w http.ResponseWriter, r *http.Request) {
 	cluster := vars["cluster"]
 	service := vars["service"]
 
-	if cluster == "" {
-		handleError(w, apierror.New(apierror.ErrNotFound, "cluster cannot be empty", nil))
-		return
-	}
-
-	if service == "" {
```

Comment:
Why did you remove these checks, are they still needed?

Reply:
It's not possible to get to the handler without those being set since they are defined in the routes